### PR TITLE
Add GeoJSON viewer, GeoDiffr and routing best practices blog entries

### PR DESCRIPTION
**Pull request by sgreenbury**
@dabreegster: as discussed I've copied and edited the GeoJSON viewer, GeoDiffr and routing best practices from your blog post - feel free to edit/add to any of this, thanks!

### CODE REVIEW

File: src/content/blogs/platform_v2.md [hunk 1]
```diff
@@ -115,6 +116,25 @@ To make it easier to work with SPC outputs, we have added two new features:
...
 
+#### New tools for working with GeoJSON
+
+![gj_crossings](/blog_content/v2_release/gj_crossings.gif)
+
+Developed partly during Turing UA hackathon days, two exciting new "day-to-day" tools now make it easier to work with GeoJSON files:
+
+- [GeoJSON viewer](https://dabreegster.github.io/geojson-viewer): aiming to fill the gap between [geojson.io](https://geojson.io/) (missing features such as filtering, slow for large input) and QGIS (overpowered and hard to use), GeoJSON viewer empowers users with a fast viewer allowing them to filter and colour by attributes to spot real examples of features quickly.
+- [GeoDiffr](https://dabreegster.github.io/geodiffr): enabling users to compare two GeoJSON files and visually identify the differences between them. Example use cases include comparing the outputs of different routing algorithms or exploring how areas of interest have changed over time in OSM.
```
Comment (dabreegster) [inline src/content/blogs/platform_v2.md]:
```suggestion
- [GeoDiffr](https://dabreegster.github.io/geodiffr): enabling users to compare two GeoJSON files and visually identify the differences between them. Example use cases include comparing the outputs of different routing algorithms or exploring how areas of interest have changed over time in OpenStreetMap.
```

(OSM might be unfamiliar acronym)

File: src/content/blogs/platform_v2.md [hunk 2]
```diff
@@ -183,4 +203,8 @@ On the tooling side of Popgetter, we are planning on building a number of differ
 want from Popgetter, we are planning to build out a terminal user interface and web interface. We also think there is great utility in making Popgetter available in the data science and web tooling
 contexts. To enable those use cases, we will be developing Python and JavaScript interfaces for the Popgetter library.
 
-####
+#### Routing best practices: a guide
```
Comment (dabreegster) [inline src/content/blogs/platform_v2.md]: I'd maybe leave this one out. If somebody clicks through to the guide right now, they'll just see a confusing WIP thing. I'd vote to just not mention it until it's minimally readable

**Review by dabreegster — APPROVED** [review]
Thanks so much for writing these up!

File: src/content/blogs/platform_v2.md (hunk 2 above)
Comment (sgreenbury) [reply src/content/blogs/platform_v2.md]: Sounds good, I'll go ahead and remove this section.

**Comment by sgreenbury** [comment]
Thanks @dabreegster! Merging this now.

**Pull request merged**